omrmakefiles/rules.aix.mk

```diff
@@ -41,6 +41,16 @@ GLOBAL_CFLAGS += -q mbcs -qlanglvl=extended -qarch=ppc -qinfo=pro -qalias=noansi
 GLOBAL_CXXFLAGS+=-q mbcs -qlanglvl=extended -qarch=ppc -qinfo=pro -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108
 GLOBAL_CPPFLAGS+=-D_XOPEN_SOURCE_EXTENDED=1 -D_ALL_SOURCE -DRS6000 -DAIXPPC -D_LARGE_FILES
 
+ifdef I5_VERSION
+  I5_FLAGS+=-g -qtbtable=full -qlist -qsource 
+  I5_DEFINES+=-DJ9OS_I5 -DJ9OS_$(I5_VERSION) -I$(top_srcdir)/../iseries -I$(top_srcdir)/../oti
```

Comment:
This line looks J9 Java specific.  If that is the case the code should be made in the J9 Java language extensions.